cloudigrade/config/settings/base.py

```diff
@@ -448,15 +448,48 @@ def __print(*args, **kwargs):
 
 CELERY_TASK_ROUTES = {
     # api.tasks
-    "api.tasks.delete_cloud_account": {"queue": "delete_cloud_account"},
-    "api.tasks.delete_inactive_users": {"queue": "delete_inactive_users"},
-    "api.tasks.inspect_pending_images": {"queue": "inspect_pending_images"},
-    "api.tasks.persist_inspection_cluster_results_task": {
-        "queue": "persist_inspection_cluster_results_task"
+    "api.tasks.calculate_max_concurrent_usage": {
+        "queue": "calculate_max_concurrent_usage"
+    },
+    "api.tasks.delete_cloud_account": {
+        "queue": "delete_cloud_account"
+    },
```

Comment:
Do you have your IDE or other local tooling configured to run `black` automatically against your code?

On my system, `black` removes excess whitespace and collapses lines like this, and the result looks like the previous version of the file like this:

```
    "api.tasks.delete_cloud_account": {"queue": "delete_cloud_account"},
```

I wonder if there's something odd about our `tox` config that isn't catching this. 🤔

Reply:
Well, yep, I just discovered that we might be a bit too aggressive with our ignore pattern in `tox.init`. I may address this in a separate PR.

Reply:
My comments will be resolved soon with https://github.com/cloudigrade/cloudigrade/pull/1087